__tests__/endpoint.test.js

```diff
@@ -174,4 +174,56 @@ describe('/api/articles', () => {
         })
     })
     
-})
\ No newline at end of file
+})
+describe('/api/articles/:article_id/comment', () => {
+    
+    test("Should return a array with the properties author ,title, article_id, body. topic, created_at,  votes, article_img_url", () => {
+            return request(app).get("/api/articles/1/comments").expect(200).then(response => {
+
+                const responseObj = response.body.comment
+
+                expect(responseObj.length).toBe(11);
+
+                responseObj.map(obj=>{
+                  for (const key in obj) {
+  
+                  expect(obj).toBeInstanceOf(Object);
+                  expect(obj).toHaveProperty("comment_id", expect.any(Number));
+                  expect(obj).toHaveProperty("body", expect.any(String));
+                  expect(obj).toHaveProperty("article_id", expect.any(Number));
+                  expect(obj).toHaveProperty("votes", expect.any(Number));
+                  expect(obj).toHaveProperty("created_at", expect.any(String));
+                  }
```

Comment:
it should also have the author of the comment here too - please make sure to check for that
- also, consider if you need this to be a map - which creates a new array - is there another array method that you could use to check what is inside each element?